backend/src/main/java/com/francislainy/so/backend/controller/question/QuestionQueryController.java

```diff
@@ -16,11 +18,18 @@
 @CrossOrigin
 @RequestMapping("/api/so/questions")
 @RestController
+@Slf4j
 public class QuestionQueryController {
 
     @Autowired
     private QuestionQueryService questionQueryService;
 
+    @ResponseStatus(value = HttpStatus.FORBIDDEN)
+    @ExceptionHandler(WrongUserException.class)
+    @ResponseBody
+    public void handleWrongUserException(Exception ex) {
+        log.info("Forbidden exception throw: " + ex);
+    }
```

Comment:
This is just one way to handle different error responses. The main doc is here https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-error-handling